sdk/keyvault/test-resources.json

```diff
@@ -187,7 +194,8 @@
                 "enablePurgeProtection": false,
                 "enableSoftDelete": true,
                 "publicNetworkAccess": "Enabled",
-                "networkAcls": "[variables('networkAcls')]"
+                "networkAcls": "[variables('networkAcls')]",
+                "softDeleteRetentionInDays": 7
```

Comment:
Not sure whether this is retention for the MHSM itself or the resources it contains, but I would think we want the smallest value here.

Reply:
According to [documentation](https://docs.microsoft.com/azure/key-vault/managed-hsm/best-practices#turn-on-recovery-options) it looks like we can only set this between 7 and 90 days unfortunately

Reply:
See how I assumed the smallest value is less than 7 😄